test/unittests/mongoBackend/mongoSubscribeContext_test.cpp

```diff
@@ -331,7 +330,7 @@ TEST(mongoSubscribeContext, Ent1_Attr0_C1)
     EXPECT_EQ(id, sub.getField("_id").OID().toString());
     EXPECT_EQ(1360236300, sub.getIntField("expiration"));
     EXPECT_FALSE(sub.hasField("lastNotification"));
-    EXPECT_FALSE(sub.hasField("throttling"));
+    EXPECT_EQ(-1, sub.getIntField("throttling"));
```

Comment:
So, throttling is always present, but a value of 0 or -1 means throttling is 'off'.

Sounds ok.